inc/utility/namespace.php

```diff
@@ -45,7 +45,7 @@ function scandir_by_date( $dir ) {
 
 	$files = [];
```

Comment:
Change code comment from:
`/* Scan a directory and return the files ordered by date, newest first.` 

To:
`/* Scan a directory and return the files (not directories!) ordered by date, newest first.`

_I guess this is what we always assumed, but since this function is named scandir better to be explicit._

Change unit test`\UtilityTest::test_scandir_by_date`, add 

`$this->assertNotContains( 'data', $files );`

Where `data` is a directory in the unit test folder.